inventory/environ.py

```diff
@@ -110,6 +110,8 @@ def getDefaultVars():
     defaultVars["splunk"]["role"] = os.environ.get('SPLUNK_ROLE', 'splunk_standalone')
     defaultVars["splunk_home_ownership_enforcement"] = False if os.environ.get('SPLUNK_HOME_OWNERSHIP_ENFORCEMENT', "").lower() == "false" else True
     defaultVars["hide_password"] = True if os.environ.get('HIDE_PASSWORD', "").lower() == "true" else False
+    defaultVars["dmc_forwarder_monitoring"] = os.environ.get('DMC_FORWARDER_MONITORING', False)
+    defaultVars["dmc_asset_interval"] = os.environ.get('DMC_ASSET_INTERVAL', '3,18,33,48 * * * *')
```

Comment:
Make sure to add these two variables to the docs and document the cron schedule for the DMC assets.  15 minutes seems like a pretty good interval, but I can definitely see cases where customers would not want to run it this frequently.

Reply:
15 minutes was the default option through the UI, so I wanted to keep it consistent.